wallet/src/repository/wallet/mod.rs

```diff
@@ -1353,10 +1368,6 @@ where
                         );
                     }
                 }
-            } else if state.transient_external_addresses.contains_key(&output.pkh) {
-                own_outputs.insert(output.pkh, model::OutputType::External);
-            } else if state.transient_internal_addresses.contains_key(&output.pkh) {
-                own_outputs.insert(output.pkh, model::OutputType::Internal);
```

Comment:
Is this not needed anymore?

Reply:
Nope, because the external and internal addresses are generated (as normal ones) before getting there.